src/__swaps__/screens/Swap/resources/search/utils.ts

```diff
@@ -4,6 +4,27 @@ import { SearchAsset } from '@/__swaps__/types/search';
 import { Address } from 'viem';
 import { isNativeAsset } from '@/handlers/assets';
 
+export function parseTokenSearchResults(assets: SearchAsset[]): SearchAsset[] {
```

Comment:
added a new function here that will eventually replace / consolidate the other parsers in this file once I finish moving DiscoverSearch over to the new query store.